src/components/Modals/SelectLoomBall.tsx

```diff
@@ -0,0 +1,113 @@
+import { getItemsService } from '@src/services/items.services';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import Modal from 'react-native-modal';
+import { CustomButton } from '@src/components/CustomButton';
+import { TLoomball } from '@src/types/types';
+import { SelectLoomballGrid } from '../ItemsGrid/SelectLoomballGrid';
+
+export interface iPropsSelectLoomBallModal {
+  isVisible: boolean;
+  toggleVisibilityCallback: () => void;
+  submitCallback: (loomBall: TLoomball) => void;
+}
+
+export const SelectLoomBallModal = ({
+  isVisible,
+  toggleVisibilityCallback,
+  submitCallback
+}: iPropsSelectLoomBallModal) => {
+  const [loomBall, setLoomBall] = useState<TLoomball[]>([]);
```

Comment:
This names are confusing. Please, change 'em to loomballs, setLoomballs and selectedLoomball, setSelectedLoomball.